.github/workflows/qc.yml

```diff
@@ -28,10 +33,35 @@ jobs:
         target: 'x86_64'
         manylinux: auto
         args: --release --out dist -i ${{ matrix.python-version }}
-    - name: Install built wheel and run Python tests
+    - name: Install built wheel and run all tests (Rust and Python)
       run: |
+        # Set up
+
         pip install semsimian --no-index --find-links dist --force-reinstall
-        pip install pytest
-        # pytest
-    - name: Run Rust tests (cargo test)
-      run: cargo test
\ No newline at end of file
+        pip install pytest pytest-cov
+        cargo install cargo-llvm-cov
+        cargo llvm-cov show-env --export-prefix
+        cargo llvm-cov clean --workspace
+        
+        # The following commands run all rust tests, just like cargo test
+        # The first version emits a report for further parsing
+        # The second provides a plain-text report to the stdout
+        # cargo llvm-cov --lcov --output-path coverage.lcov
+
+        cargo llvm-cov
```

Comment:
Is this running cargo llvm-cov three times?

Reply:
Nope - the first two are calls to subfunctions and the last is the "actual" report generation (though in this case it just prints out without a report file)